closure/bin/build/source_test.py

```diff
@@ -57,7 +57,7 @@ def testSourceScanGoogModule(self):
 
     self.assertEqual(set(['foo']),
                      test_source.provides)
-    self.assertEqual(set(['bar']),
+    self.assertEqual(set(['bar', 'foo', 'foobar']),
```

Comment:
I think I'd prefer you move the ES6 module test to its own unit test method. Also, you can add a specific `_TEST_ES6_MODULE_SOURCE` for that test alone.